app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -87,11 +87,14 @@ fun TopText(text: String) {
 }
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun CreateEmailField(isEmailValid: Boolean, onEvent: (LoginEvent) -> Unit, email: String) {
+fun CreateEmailField(isEmailValid: Boolean, onValueChange: (String) -> Unit, email: String) {
     TextField(
         value = email,
         onValueChange = {
-            onEvent(LoginEvent.OnEmailChanged(it))
+            onValueChange(
+                email ->
+            )
```

Comment:
Here the syntax also isn't correct. You call a lambda like a normal function, in this case: `onValueChange(it)`